modules/flowable-cmmn-engine/src/test/resources/org/flowable/cmmn/test/runtime/UserEventListenerTest.testUserEventRepetitionWithRepeatingEntryCriteriaAndMaxInstanceCountOne.cmmn

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't these definitions be in the `src/test/resources/org/flowable/cmmn/test/eventlistener` package?